lms/djangoapps/instructor/models.py

```diff
@@ -7,7 +7,7 @@ class InstructorAvailableSections(models.Model):
     Enumerates list of tabs which could be switched off on Instructor Dashboard panel
     """
     user = models.OneToOneField(User, on_delete=models.SET_NULL, related_name='instructor_dashboard_tabs',
-                                null=True, blank=True, limit_choices_to={'is_staff': True}, verbose_name = 'Instructor')
+                                null=True, blank=True, verbose_name = 'Instructor')
```

Comment:
`verbose_name = 'Instructor'` better to use `verbose_name='Instructor'` - i mean without spaces (pep8)